public/scripts/suggestion.js

```diff
@@ -0,0 +1,17 @@
+$( document ).ready(function() {
+ $('#git_url').editableSelect();
+ $('#git_url').keyup(() => {
+   let q = $("#git_url").val()
```

Comment:
I see that you don't like to add semicolon at the end of your statements. Can you please explain if this is useful?